topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -42,19 +43,28 @@ IntelliJ Platform Gradle Plugin 2.x requires the following minimal versions:
 >
 {style="note"}
 
-To apply the IntelliJ Platform Gradle Plugin to a project, add the following entry to the `plugins` block in the <path>build.gradle.kts</path> file:
-
-```kotlin
-plugins {
-  id("org.jetbrains.intellij.platform") version "%intellij-platform-gradle-plugin-version%"
-}
-```
-
+To apply the IntelliJ Platform Gradle Plugin to a project, you need to add the following:
```

Comment:
I'm not sure, but I think the user needs to add the gradle portal repository to use the plugin. If I am wrong, this can be reverted.